static/app/views/projectInstall/createProject.tsx

```diff
@@ -215,73 +234,37 @@ function CreateProject() {
     projectName !== '' &&
     (!shouldCreateCustomRule || conditions?.every?.(condition => condition.value));
 
-  const createProjectForm = (
-    <Fragment>
-      <Layout.Title withMargins>
-        {t('3. Name your project and assign it a team')}
-      </Layout.Title>
-      <CreateProjectForm
-        onSubmit={(event: React.FormEvent<HTMLFormElement>) => {
-          // Prevent the page from reloading
-          event.preventDefault();
-          frameworkSelectionEnabled ? handleProjectCreation() : createProject();
-        }}
-      >
-        <div>
-          <FormLabel>{t('Project name')}</FormLabel>
-          <ProjectNameInputWrap>
-            <StyledPlatformIcon platform={platform?.key ?? 'other'} size={20} />
-            <ProjectNameInput
-              type="text"
-              name="name"
-              placeholder={t('project-name')}
-              autoComplete="off"
-              value={projectName}
-              onChange={e => setProjectName(slugify(e.target.value))}
-            />
-          </ProjectNameInputWrap>
-        </div>
-        <div>
-          <FormLabel>{t('Team')}</FormLabel>
-          <TeamSelectInput>
-            <TeamSelector
-              name="select-team"
-              aria-label={t('Select a Team')}
-              menuPlacement="auto"
-              clearable={false}
-              value={team}
-              placeholder={t('Select a Team')}
-              onChange={choice => setTeam(choice.value)}
-              teamFilter={(filterTeam: Team) => filterTeam.hasAccess}
-            />
-            <Button
-              borderless
-              data-test-id="create-team"
-              icon={<IconAdd isCircled />}
-              onClick={() =>
-                openCreateTeamModal({
-                  organization,
-                  onClose: ({slug}) => setTeam(slug),
-                })
-              }
-              title={t('Create a team')}
-              aria-label={t('Create a team')}
-            />
-          </TeamSelectInput>
-        </div>
-        <div>
-          <Button
-            type="submit"
-            data-test-id="create-project"
-            priority="primary"
-            disabled={!canSubmitForm}
-          >
-            {t('Create Project')}
-          </Button>
-        </div>
-      </CreateProjectForm>
-    </Fragment>
-  );
+  const alertFrequencyDefaultValues = useMemo(() => {
```

Comment:
this new function checks for the alert saved in the context, converting it to default value formats used in the `IssueAlertOptions` component.